framework/src/eventloop/eventloop_task.c

```diff
@@ -40,30 +39,36 @@ int get_async_task(void)
 
 int eventloop_task(int argc, char *argv[])
 {
-	el_loop_t *loop;
 	sigset_t sigset;
+	el_handle_t *handle;
+	el_app_handle_t *app_handle;
 
-	loop = uv_default_loop();
-	if (loop == NULL) {
-		eldbg("Failed to get default loop\n");
-		return -1;
-	}
-
-	/* Unset all signals except for SIGEL_WAKEUP and SIGKILL */
+	/* Unset all signals except for SIGEL_WAKEUP */
 	sigfillset(&sigset);
 	sigdelset(&sigset, SIGEL_WAKEUP);
-	sigdelset(&sigset, SIGKILL);
 	(void)sigprocmask(SIG_SETMASK, &sigset, NULL);
 
+	app_handle = (el_app_handle_t *)eventloop_get_app_handle();
```

Comment:
if return of the eventloop_get_app_handle is NULL, is it okay?

Reply:
yes. it is not NULL definitely because it is address of space defined as global array.